plugins/woocommerce/tests/e2e-pw/tests/merchant/products/add-variable-product/update-variations.spec.js

```diff
@@ -138,14 +138,12 @@ test.describe( 'Update variations', () => {
 		} );
 
 		await test.step( 'Click on the "Variations" tab.', async () => {
-			await page.locator( 'a[href="#variable_product_options"]' ).click();
+			await page.getByRole('link', { name: 'Variations' }).click();
```

Comment:
Playwright was occasionally failing with the previous locator, so updating to a `getByRole()`

Reply:
That's actually what's expected.  I used the Playwright locator selector, and it gave me that string.  If you drop the character, there are multiple elements matched.  That character is actually the icon displayed in that menu item (maybe in the future, that will be refactored to an SVG or something)